js/ui/password_picker/password_picker.js

```diff
@@ -281,10 +290,15 @@ $(document).ready(function () {
             disablePassman('url', tab.url);
         });
 
+        if (IsMasterPasswordSet === false) {
+            makeTabActive('unlock');
+        }
+
         API.runtime.sendMessage(API.runtime.id, {
```

Comment:
I think this sendMessage command is sometimes useless and only required in the above else case

Reply:
You are right. I've moved sendMessage into else-block:
```
        if (IsMasterPasswordSet === false) {
            makeTabActive('unlock');
        } else {
            API.runtime.sendMessage(API.runtime.id, {
                method: "getCredentialsByUrl",
        // ...
        }
```